src/UI/Root/Root.test.tsx

```diff
@@ -1,138 +1,97 @@
 import React, { act } from "react";
-import { useLocation } from "react-router";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { render, screen } from "@testing-library/react";
 import { StoreProvider } from "easy-peasy";
 import { axe, toHaveNoViolations } from "jest-axe";
-import { Either } from "@/Core";
-import {
-  QueryResolverImpl,
-  getStoreInstance,
-  GetServerStatusOneTimeQueryManager,
-  GetServerStatusStateHelper,
-  GetEnvironmentsStateHelper,
-  GetEnvironmentsQueryManager,
-} from "@/Data";
+import { HttpResponse, http } from "msw";
+import { setupServer } from "msw/node";
+import { FileFetcherImpl, getStoreInstance } from "@/Data";
 import { defaultAuthContext } from "@/Data/Auth/AuthContext";
-import {
-  GetEnvironmentsContinuousQueryManager,
-  GetEnvironmentsContinuousStateHelper,
-} from "@/Data/Managers/GetEnvironmentsContinuous";
 import {
   DeferredApiHelper,
-  dependencies,
-  DynamicQueryManagerResolverImpl,
+  Environment,
+  MockedDependencyProvider,
   Project,
   ServerStatus,
-  StaticScheduler,
 } from "@/Test";
-import { DependencyProvider, EnvironmentHandlerImpl } from "@/UI/Dependency";
 import { TestMemoryRouter } from "../Routing/TestMemoryRouter";
 import { Root } from "./Root";
 
+jest.spyOn(defaultAuthContext, "getToken").mockReturnValue("mocked_token");
```

Comment:
Hmmm, it didn't require mocking before, is this an unwanted side-effect that needs to be checked out?